src/cursor/cur_file.c

```diff
@@ -298,6 +407,7 @@ __wt_curfile_insert_check(WT_CURSOR *cursor)
     cbt = (WT_CURSOR_BTREE *)cursor;
     tret = 0;
     CURSOR_UPDATE_API_CALL_BTREE(cursor, session, update);
+    F_CLR(cbt, WT_CBT_RESEARCH);
```

Comment:
We have to use something other than `research`, that's a word, I can't read it any other way than "research". :)

`reposition`?

Reply:
OK